src/custom/components/TransactionConfirmationModal/index.tsx

```diff
@@ -152,12 +152,12 @@ const ButtonCustom = styled.button`
   }
 `
 
-const CheckCircleCustom = styled(CheckCircle)`
-  height: auto;
-  width: 20px;
-  max-height: 100%;
-  margin: 0 10px 0 0;
-`
+// const CheckCircleCustom = styled(CheckCircle)`
+//   height: auto;
+//   width: 20px;
+//   max-height: 100%;
+//   margin: 0 10px 0 0;
+// `
```

Comment:
This file is not a mod, is it?
I would be in favor of removing rather than commenting out

Reply:
I'm not sure I follow. 
As far as I can see its a mod.

Reply:
I mean, the `*Mod` file in the same folder is a mod, this is the index file which is completely owned by us.
I'm in favor of removing stuff from this one rather than commenting out

Reply:
Ok, removed comments